spell-check-word-allow-list.yaml

```diff
@@ -0,0 +1,162 @@
+whiteList:
+  - nodoc
+  - Luhn
+  - luhn
+  - gregorian
+  - kcp
+  - Prefilled
+  - prefill
+  - docs.adyen.com
+  - Cartebancaire
+  - cartebancaire
+  - americanExpress
+  - Forbrugsforeningen
+  - visadankort
+  - dankort
+  - Sdk
+  - sdk
+  - bcmc
+  - binlookup
+  - plcc
+  - uatp
+  - nyce
+  - Dejeneur
+  - cencosud
+  - accel
+  - jcb
+  - hipercard
+  - codensa
+  - argencard
+  - maestrouk
+  - karenmillen
+  - hipercard
+  - visaalphabankbonus
+  - mcalphabankbonus
+  - Indomaret
+  - interac
+  - Dotpay
+  - Qiwi
+  - qiwi
+  - qiwiwallet
+  - boletobancario
+  - applepay
+  - wechatpay
+  - mbway
+  - bancontact
+  - econtext
+  - Bancairo
+  - directdebit
+  - sepa
+  - oxxo
+  - genericgiftcard
+  - openbanking
+  - molpay
+  - paywithgoogle
+  - afterpay
+  - androidpay
+  - amazonpay
+  - santander
+  - hpp
+  - Doku
+  - Boleto
+  - Multibanco
+  - giropay
+  - Palx
+  - klarna
+  - ebanking
+  - elodebit
+  - swiftlint
+  - Hashable
+  - Codable
+  - Hidable
+  - rhs
+  - lhs
+  - Unkeyed
+  - psp
+  - Iterable
+  - Finalizable
+  - GitHub
+  - weixin
+  - Eldoheiri
+  - Abramichev
+  - mohamed
+  - qr
+  - katrina@mail
+  - pspreference
+  - authorisation
+  - sut
+  - \n
+  - tx
+  - checkoutshopperios
+  - explicit_acl
+  - cyclomatic_complexity
+  - korean
+  - korea
+  - Heimdall
+  - authorise
+  - associatedtype
+  - threeds
+  - acl
+  - xcrun
+  - ios
+  - validatable
+  - Tintable
+  - Diffable
+  - uts
+  - uname
+  - lproj
+  - autoclosure
+  - fputs
+  - unobserve
+  - Unobserve
+  - checkoutshopper
+  - Btn
+  - btn
+  - adyen
+  - Adyen
+  - Ady
+  - enc
+  - Cryptor
+  - hmac
+  - Hmac
+  - Attr
+  - henrinormak
+  - stackoverflow
+  - com
+  - threeds2
+  - Enc
+  - Ephem
+  - kty
+  - crv
+  - fingerprint
+  - 3ds2
+  - Initializable
+  - app
+  - id
+  - noncestr
+  - exp
+  - rgb
+  - autoreleasepool
+  - Prefilling
+  - yy
+  - utsname
+  - uiactivityviewcontroller
+  - appid
+  - partnerid
+  - prepayid
+  - Taxno
+  - Heimdall.swift
+  - Shopperi
+  - fpx
+  - suica
+  - quic
+  - eftpos
+  - nanaco
+  - waon
+  - deinit
+  - darwin
+  - cyclomatic
+  - doesn
+  - couldn
+  - brazilian
+  - australian
```

Comment:
What is wrong here? Should be capital?

Reply:
its a quirk of the spell check tool, some how it thinks `Australian` and `Brazilian` are typos 🤷‍♂️